### PR TITLE
Task/screenly conf file

### PR DESCRIPTION
**Pull request by rusko124**
After migration from master to experimental, server and viewer were failed because they misidentified the paths from `screenly.conf`

### CODE REVIEW

File: ansible/roles/screenly/tasks/main.yml [hunk 1]
```diff
@@ -9,6 +9,14 @@
     - .config
     - screenly_assets
 
+- name: Copy Screenly default config
```
Comment (vpetersson) [inline ansible/roles/screenly/tasks/main.yml:12]: We can remove this block entirely. It's done [here](https://github.com/Screenly/screenly-ose/blob/experimental/bin/start_server.sh#L8-L10).

**Comment by rusko124** [comment]
@vpetersson updated

**Comment by vpetersson** [comment]
You shouldn't need this. I moved this to [here](https://github.com/Screenly/screenly-ose/blob/experimental/bin/start_server.sh).

**Comment by vpetersson** [comment]
@rusko124 is this still relevant?

**Comment by rusko124** [comment]
@vpetersson No, we can close it, but we need to add a similar code to a migrate script.

**Pull request closed without merging**